WebGL.md

```diff
@@ -0,0 +1,65 @@
+# Implementing Unity-Immutable-SDK for WebGL
+
+This guide provides focused instructions for implementing the Unity-Immutable-SDK in WebGL projects.
+
+For general information on the SDK, please refer to
+the [Unity-Immutable-SDK documentation](https://docs.immutable.com/sdks/zkEVM/unity).
+
+Live example can be found at https://immutable.github.io/unity-immutable-sdk/sample/webgl
+
+## WebGL Template Setup
+
+1. Navigate to `sample/Assets/WebGLTemplates/unity-webview/`.
+
+2. Copy the following files to your WebGLTemplates folder:
+   - `unity-webview.js`
+   - `callback.html`
+   - `logout.html`
+   - `Passport/index.html`
```

Comment:
It's weird to me that the users need to copy files from the sample app to make it work. Or is this only for now? Will this change when you make it publicly available? 

For example, create template files the user could copy and paste (like custom Windows webview solution code [here](https://docs.immutable.com/sdks/zkEVM/unity/custom-windows-webview)). There might be other better solutions.

Reply:
I'm also not sure about it, remember I was putting these templates inside the package?
Let me think about it and get back to you.